tests/Neo.Extensions.Tests/UT_ByteExtensions.cs

```diff
@@ -25,7 +25,7 @@ public class UT_ByteExtensions
         public void TestToHexString()
         {
             byte[] nullStr = null;
-            Assert.ThrowsException<NullReferenceException>(() => nullStr.ToHexString());
+            Assert.ThrowsException<ArgumentNullException>(() => nullStr.ToHexString());
```

Comment:
does this mean Net9.0 has changed the exception type?

Reply:
yes

Reply:
> does this mean Net9.0 has changed the exception type?

`ToHexString` uses `Convert.ToHexStrongLower` when run with NET.9.
May need to add an argument null-check to keep same semantics.